services/movie_session.py

```diff
@@ -0,0 +1,44 @@
+from datetime import datetime
+from db.models import MovieSession
+
+
+def create_movie_session(
+    movie_show_time: str, movie_id: int, cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id,
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> MovieSession:
```

Comment:
The return type hint for `get_movies_sessions` should be `QuerySet` instead of `MovieSession` because the function returns a QuerySet of `MovieSession` instances, not a single instance.